scheds/rust/scx_layered/src/bpf/main.bpf.c

```diff
@@ -2667,6 +2687,21 @@ static void refresh_cpus_flags(struct task_ctx *taskc,
 			break;
 		}
 	}
+	if (enable_cpuset) {
+		bpf_for(cpuset_id, 0, nr_cpusets) {
+			struct cpumask_wrapper* wrapper;
+			wrapper = bpf_map_lookup_elem(&cpuset_cpumask, &cpuset_id);
+			if (!wrapper || !wrapper->mask) {
+				scx_bpf_error("error marking tasks as cpuset aligned");
+				return;
+			}
+			if (bpf_cpumask_equal(cast_mask(wrapper->mask), cpumask)) {
+				taskc->cpus_cpuset_aligned = true;
+				return;
```

Comment:
`break;` here so that it's consistent with the node aligned block and the function can be extended in the future? Note that this would require moving the `false` setting. BTW, why not use the same partial overlapping test used by node alignment test instead of equality test? Is that not sufficient for forward progress guarantee? If not, it'd probably be worthwhile to explain why.

Reply:
Ahh, correct me if I am wrong, that partial test is wrong/the node code should do what the cpuset code does?

For example:
```
1000 task_mask
1100 system_mask

system_mask intersect task_mask == true
system_mask subset task_mask == false
```

IIUC, allow node aligned is only supposed to avoid lo fb when task cpumask matches node cpumasks to ensure lo fb is used for tasks with single cpu affinities (i.e. which are stall prone)?